ts/twitter/.env.sample

```
@@ -1,2 +1,4 @@
 NEXT_PUBLIC_LOCAL_API_BASE_URL=http://localhost:80
 NEXT_PUBLIC_USER_ID=
+RABBITMQ_URL=amqp://guest:guest@rabbitmq:5672
```

Sending events, such as impressions or clicks, is entirely handled on the client side.  
In Next.js, environment variables that are accessed from the client (not just the frontend server) must be prefixed with `NEXT_PUBLIC`.

```suggestion
NEXT_PUBLIC_RABBITMQ_URL=amqp://guest:guest@rabbitmq:5672
```